cli/i18n/I18nData.php

```diff
@@ -182,7 +182,16 @@ public function addValue($key, $value, $language) {
 		    !array_key_exists($key, $this->data[static::REFERENCE_LANGUAGE][$this->getFilenamePrefix($key)])) {
 			throw new Exception('The selected key does not exist for the selected language.');
 		}
-		$this->data[$language][$this->getFilenamePrefix($key)][$key] = $value;
+		if (static::REFERENCE_LANGUAGE === $language) {
+			$previousValue = $this->data[static::REFERENCE_LANGUAGE][$this->getFilenamePrefix($key)][$key];
```

Comment:
I think that'd be more in line with the general style, unless I'm reading more into it than there is. (As in camelCase = function, object or method. More of a question for @Alkarex.)
```suggestion
			$previous_value = $this->data[static::REFERENCE_LANGUAGE][$this->getFilenamePrefix($key)][$key];
```

Reply:
I personally favour camelCase for new code (except when in the middle of another piece of code, which uses another convention), but we do have a bit of inconsistency :-)